tasmoadmin/rootfs/etc/nginx/nginx.conf

```diff
@@ -12,42 +12,5 @@ http {
     sendfile           on;
     keepalive_timeout  65;
 
-    server {
-        server_name hassio.local;
-        listen 9541 default_server;
-        root /var/www/tasmoadmin/tasmoadmin/;
-        index index.php;
-
-        location /data/firmwares {
-        }
-
-        location /data/ {
-            deny all;
-        }
-
-        location ~ .php$ {
-            fastcgi_pass 127.0.0.1:9001;
-            fastcgi_read_timeout 900;
-            fastcgi_split_path_info ^(.+\.php)(/.+)$;
-            fastcgi_index index.php;
-            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-            include fastcgi_params;
-        }
-
-        location ~ ^(.*)\.(css|js|gif||jpe?g|png|json|cache\.json)$ {
-        }
-
-        location / {
-            rewrite ^/login$ /login.php last;
-            rewrite ^/logout$ /login.php?logout=logout last;
-            rewrite ^/doAjaxAll$ /index.php?doAjaxAll=doAjaxAll last;
-            rewrite ^/doAjax$ /index.php?doAjax=doAjax last;
-            rewrite "/([a-z]{2})/" /index.php?lang=$1 last;
-            rewrite ^/([a-zA-Z_]+)/([a-zA-Z_]+)/([0-9_]+)/?$ /index.php?page=$1&action=$2&device_id=$3;
-            rewrite ^/([a-zA-Z_]+)/(force)/?$ /index.php?page=$1&force=1;
-            rewrite ^/([a-zA-Z_]+)/([a-zA-Z_]+)/?$ /index.php?page=$1&action=$2;
-            rewrite ^/([a-zA-Z_]+)/([0-9]+)/?$ /index.php?page=$1&device_id=$2;
-            rewrite ^/([a-zA-Z_]+)/?$ /index.php?page=$1;
-        }
-    }
+    include /etc/nginx/servers/*.conf;
```

Comment:
Please use the Nginx config structure as used by other community add-ons as well.
Which is not this. Keeping things similar makes maintenance easier.

Reply:
This was how vscode did it... https://github.com/hassio-addons/addon-vscode/tree/master/vscode/rootfs/etc/nginx/servers